<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandSettingsPage</class>
 <widget class="QWidget" name="CommandSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="commandGroupBox">
     <property name="title">
      <string>Command</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioCommand_1">
        <property name="text">
         <string>Don't move the media</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioCommand_2">
          <property name="text">
           <string>Move the media to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pathLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioCommand_3">
        <property name="text">
         <string>Move the media to the System Bin</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Run the command...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRunOption_1">
          <property name="text">
           <string>immediately</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioRunOption_2">
          <property name="text">
           <string>at the end of the media</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="detachCheckBox">
        <property name="text">
         <string>Remove the media from the playlist</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
